core/templates/components/skill-selector/merge-skill-modal.component.html

```diff
@@ -7,7 +7,6 @@ <h3>
 <div class="modal-body">
   <oppia-skill-selector [sortedSkillSummaries]="skillSummaries"
                         [selectedSkillId]="selectedSkillId"
-                        [countOfSkillsToPrioritize]="countOfSkillsToPrioritize"
```

Comment:
Bit confused about this deletion, since in the next file skill-selector.component.ts you actually have it defined. So why delete it here?